editor.planx.uk/src/@planx/graph/index.ts

```diff
@@ -173,8 +174,7 @@ const _add = (
     before = undefined,
   }: { children?: Child[]; parent: string; before?: string },
 ) => {
-  if (draft[id]) throw new Error("id exists");
```

Comment:
Removing this is required when pasting a collection of nodes which include clones.

My understanding of this is that it's just a guard clause against ID clashes - if we did have a clash it would overwrite the previous node.

Our `uniqueId()` function for clones (line 28 above) has an astronomically small chance of clashing - there's ~839 quadrillion possible outputs 👀